src/amber/cli/commands/pipelines.cr

```diff
@@ -77,6 +77,7 @@ module Amber::CLI
           case line
           when .starts_with?("pipeline") then set_pipe(line)
           when .starts_with?("plug")     then set_plug(line)
+          else
```

Comment:
Should this and [other](https://github.com/amberframework/amber/pull/1201/files#diff-eb22ae110d5e9148a4e2657f50faf2e7R85) places where Crystal wants an else have a puts, raise, or something else so they don't fail silently?

Reply:
good idea

Reply:
This one actually needs to skip lines that are not pipeline so its not a failure case.